src/main/webapp/react.html

```diff
@@ -0,0 +1,53 @@
+<!--
```

Comment:
Please also call prettier on this file:

`prettier --write  src/main/webapp/react.html`